src/class_sqlThread.py

```diff
@@ -29,22 +18,144 @@
     import tr
     from bmconfigparser import BMConfigParser
     from debug import logger
-    # pylint: disable=attribute-defined-outside-init,protected-access
     from addresses import encodeAddress
+except ImportError:
+    from . import helper_sql
+    from . import helper_startup
+    from . import paths
+    from . import queues
+    from . import state
+    from . import tr
+    from .bmconfigparser import BMConfigParser
+    from .debug import logger
+    from .addresses import encodeAddress
 
+# pylint: disable=attribute-defined-outside-init,protected-access
+root_path = os.path.dirname(os.path.dirname(__file__))
 
-class sqlThread(threading.Thread):
+
+def connection_build():
```

Comment:
this should contain an argument that allows to use an in memory database instead of the file.

Reply:
Done